.github/workflows/ci.yml

```diff
@@ -186,7 +186,8 @@ jobs:
                   component/test_vpc_offerings
                   component/test_vpc_routers
                   component/test_vpn_users",
-                "component/test_vpc_network_lbrules" ]
+                "component/test_vpc_network_lbrules",
+                "component/test_router_resources"]
```

Comment:
this means it gets its own run(ner), which might not be the most economic way to do it.